### PR TITLE
Meta: Use new template form for rule proposals

### PR DESCRIPTION
**Pull request by fregante**
Live example here: https://github.com/fregante/sandbox/issues/new?assignees=&labels=new+rule%2Cevaluating&template=unicorn.yml&title=Rule+proposal%3A+

## Screenshot

Screen Shot

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/rule_proposal.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      label: Description
+      description: Explain here why this rule would be beneficial
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Fail
+      description: Specify an example of code that should be detected
+      value: |
+        ```ts
```
Comment (sindresorhus) [inline .github/ISSUE_TEMPLATE/rule_proposal.yml:26]:
```suggestion
        ```js
```

?

Comment (fregante) [reply .github/ISSUE_TEMPLATE/rule_proposal.yml:26]:
```suggestion
        ``` JavaScript
```

😌

**Review by sindresorhus — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/rule_proposal.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      required: true
+    attributes:
+      label: Description
+      description: Explain here why this rule would be beneficial
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Fail
+      description: Specify an example of code that should be detected
```
Comment (fisker) [inline .github/ISSUE_TEMPLATE/rule_proposal.yml:24]:
```suggestion
      description: Specify examples of code that should be detected
```

Normally, the examples should be more than one.

Comment (fisker) [reply .github/ISSUE_TEMPLATE/rule_proposal.yml:24]: I think it's even better if we add another code block in `value`

**Comment by sindresorhus** [comment]
Bump :)

**Comment by fregante** [comment]
Bumpification successful

**Pull request merged**